extensions/integration-tests/src/utils.ts

```diff
@@ -36,3 +36,37 @@ export async function connectToServer(server: TestServerProfile, timeout: number
 export async function ensureConnectionViewOpened() {
 	await vscode.commands.executeCommand('dataExplorer.servers.focus');
 }
+
+export async function sleep(ms: number): Promise<{}> {
+	return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+export async function createDB(dbName: string, ownerUri: string): Promise<void> {
+	let queryProvider = azdata.dataprotocol.getProvider<azdata.QueryProvider>('MSSQL', azdata.DataProviderType.QueryProvider);
```

Comment:
I'm surprised we don't have a central library for this. We should consider creating one of one doesn't exist.